.readthedocs.yml

```diff
@@ -15,7 +15,7 @@ formats: all
 build:
   os: ubuntu-22.04
   tools:
-    python: "3.11"
+    python: "3.11"  # This Python version should be excluded from `.github/workflows/tests.yml`.
```

Comment:
no. The tests should run under this version `tox -e py311`.
But the `tox -e docs` does not generate coverage information, this it is excluded for now.
I would revert this.

Reply:
Cool! Thanks for reviewing! I think, if you have any further questions, let me know!

Reply:
Same here. This is fine.